src/components/time-ago.tsx

```diff
@@ -1,22 +1,14 @@
 import { useState } from 'react'
-import ReactTimeAgo from 'timeago-react'
+import ReactTimeAgo, { TimeAgoProps as ReactTimeAgoProps } from 'timeago-react'
 import * as timeago from 'timeago.js'
 
 import { useInstanceData } from '@/contexts/instance-context'
 
-interface TimeAgoProps {
-  datetime: timeago.TDate
-  opts?: timeago.Opts
-  className?: string
+interface TimeAgoProps extends Pick<ReactTimeAgoProps, 'datetime'> {
   dateAsTitle?: boolean
 }
 
-export function TimeAgo({
-  datetime,
-  opts,
-  className,
```

Comment:
Please check if removing `className` breaks custom styled components (e.g. in notification.tsx we use a `<StyledTimeAgo/>`). This is the reason it was added.

Reply:
Is there a way to try the login feature locally?

Reply:
sure, if you don't change the default setup, you should be able to login (via redirect to staging).

Reply:
@elbotho Yes, you are right. Parameter `className` is needed. I have added it again as a paramater.